client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/style.scss

```diff
@@ -112,6 +112,8 @@
 		width: 280px;
 
 		.components-popover__content {
+			max-height: none !important;
+			max-width: 100% !important;
```

Comment:
This is required due to some rendering quirks in the version of `<Popover />` in the version of `@wordpress/components` that WCS&T uses - on smaller screens the popover gets into a resizing loop.